tsconfig-lint.json

```diff
@@ -111,8 +111,7 @@
   "files": [
     "assets/constants.ts",
     "assets/rich_text_components_definitions.ts",
-    "src/main.ts",
-    "core/tests/karma.conf.ts"
```

Comment:
Leaving a note to make sure we remember to restore this before merging

Reply:
Reverted.